backend/lm_backend/api/routes/configurations.py

```diff
@@ -149,15 +151,93 @@ async def read_configuration(
 )
 async def update_configuration(
     configuration_id: int,
-    configuration_update: ConfigurationUpdateSchema,
+    configuration_update: ConfigurationCompleteUpdateSchema,
     secure_session: SecureSession = Depends(secure_session(Permissions.CONFIG_EDIT)),
 ):
-    """Update a configuration in the database."""
-    return await crud_configuration.update(
-        db_session=secure_session.session,
-        id=configuration_id,
-        obj=configuration_update,
-    )
+    """
+    Update a configuration in the database.
+
+    If there are features in the payload, they'll be updated if they have an id,
+    or will be created if the id is None.
```

Comment:
For a compound sentence, you need to have a subject in the second independent clause. Alternatively, you could make the sentence have a compound predicate instead by removing the comma:

Compound sentence:
```suggestion
    If there are features in the payload, they'll be updated if they have an id,
    or they will be created if the id is None.
```

Compound predicate:
```suggestion
    If there are features in the payload, they'll be updated if they have an id
    or will be created if the id is None.
```

Reply:
Thanks, I updated it to a compound sentence.